.travis.yml

```diff
@@ -1,6 +1,6 @@
 before_install:
   - gem update bundler
-  - sed -i '1d' Gemfile
+  - echo "RUBY_VERSION: $RUBY_VERSION"
```

Comment:
I restarted the builds, but they still fail. This time it appears to be because of a string quoting issue on this line.
```
$ {:"echo \"RUBY_VERSION"=>"$RUBY_VERSION\""}
/home/travis/build.sh: line 45: {:echo "RUBY_VERSION=: command not found
```
To fix it